proto/regen/ecocredit/v1/query.proto

```diff
@@ -150,7 +150,7 @@ message QueryClassInfoRequest {
 message QueryClassInfoResponse {
 
   // info is the ClassInfo for the credit class.
-  ClassInfo info = 1;
+  Class info = 1;
```

Comment:
I think this should be `class` rather than `info` but this will be addressed in #1022.